ts-packages/web/src/app/spaces/[id]/_components/dropdown/dropdown-menu.tsx

```
@@ -0,0 +1,100 @@
+      case ' ':
+        e.preventDefault();
+        action();
+        onclose();
+        break;
+      case 'Escape':
+        e.preventDefault();
+        onclose();
+        break;
+      case 'ArrowDown':
+        e.preventDefault();
+        const nextItem = e.currentTarget.nextElementSibling as HTMLElement;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)